app/hypercerts/[hypercertId]/page.tsx

```diff
@@ -47,17 +52,32 @@ export default async function HypercertPage({ params, searchParams }: Props) {
   const { hypercertId } = params;
 
   const [hypercert] = await Promise.all([getHypercert(hypercertId)]);
```

Comment:
I know this is not a part of this review, but what is going on here? Why is this in a `Promise.all` and then destructured again in the assignment?

Reply:
I think it's an artefact from when we split up the data fetching, pushed the fix

Reply:
We can actually remove this for a simple call whether the hypercert ID is known, or remove it completely. The components each do their own fetching